README.md

```diff
@@ -16,7 +16,7 @@ To get in touch with other Bay Area mappers and the broader U.S. mapper communit
 
 [Code for San José](https://www.codeforsanjose.org/) (CfSJ) is a nonprofit community organization and a member of the national Code for America network. CfSJ hosts a biweekly Map Night for South Bay mappers. Upcoming Map Nights are listed on [Meetup](https://www.meetup.com/Code-for-San-Jose/) and [OSMCal](https://osmcal.org/?in=United%20States) and are currently online-only due to the pandemic.
 
-To get in touch with other South Bay mappers and civic tech volunteers, invite yourself to the [Code for San José Slack](https://join.slack.com/t/codeforsanjose/shared_invite/zt-iwnx99kh-motBC0J47O8ItlR3zjAIBA) workspace and join the [#proj-osm](https://codeforsanjose.slack.com/archives/C69JM3UDD) channel. If you have a cool idea for something local mappers could add to OpenStreetMap, please [open a new issue](https://github.com/codeforsanjose/OSM-SouthBay/issues/new/).
+To get in touch with other South Bay mappers and civic tech volunteers, invite yourself to the [Code for San José Slack](https://join.slack.com/t/codeforsanjose/shared_invite/zt-iwnx99kh-motBC0J47O8ItlR3zjAIBA) workspace and join the [#proj-osm](https://codeforsanjose.slack.com/archives/C69JM3UDD) channel, or join the [Code for San José Discord](https://discord.gg/wNZpgCEYDu) server and join the #🗺️openstreetmap channel. If you have a cool idea for something local mappers could add to OpenStreetMap, please [open a new issue](https://github.com/codeforsanjose/OSM-SouthBay/issues/new/).
```

Comment:
For those who are already in the server, since the channel name has an emoji in it:

```suggestion
To get in touch with other South Bay mappers and civic tech volunteers, invite yourself to the [Code for San José Slack](https://join.slack.com/t/codeforsanjose/shared_invite/zt-iwnx99kh-motBC0J47O8ItlR3zjAIBA) workspace and join the [#proj-osm](https://codeforsanjose.slack.com/archives/C69JM3UDD) channel, or join the [Code for San José Discord](https://discord.gg/wNZpgCEYDu) server and join the [#🗺️openstreetmap](https://discord.com/channels/1086028270455230625/1086348869329227877) channel. If you have a cool idea for something local mappers could add to OpenStreetMap, please [open a new issue](https://github.com/codeforsanjose/OSM-SouthBay/issues/new/).
```